supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -228,6 +228,22 @@ When combined with privileges, permissions define a user's or resource's overall
 
 *See also*:
 
+[[playbook]]
+==== image:images/yes.png[yes] playbook (noun)
+*Description*: Playbooks are Ansible's configuration, deployment, and orchestration language.
+Playbooks can describe a policy you want your remote systems to enforce, or a set of steps in a general IT process.
+When using the term _playbook_ without the Ansible prefix, use lowercase _p_.
+
+Examples:
+
+* Run a playbook in Ansible.
```

Comment:
```suggestion
* Run a playbook in Ansible.
* Run an Ansible Playbook.
```
- Consider adding an additional example so that you show both acceptable usages.